### ISSUE TITLE
Unused type ignore comment are not displayed

### ISSUE THREAD

**Issue by Morikko** [matangover/mypy-vscode#85]
By using this code:

```python
toto: int = "toto"
tata: int = 5  # type: ignore
```

The output of `Mypy Type Checker` is:

```
2024-01-24 11:20:54.152 [info] file:///path/to/file/test.py :
/path/to/file/test.py:69:13:69:18: error: Incompatible types in assignment (expression has type "str", variable has type "int")  [assignment]
/path/to/file/test.py:70: error: Unused "type: ignore" comment  [unused-ignore]
```

But only the first line has a visual feedback:

![image](https://github.com/matangover/mypy-vscode/assets/7831572/60e220e5-2371-4049-9dea-75bbc159b4ce)

I suspect that the missing column may be the root cause as it is creating [an empty range in the code](https://github.com/matangover/mypy-vscode/blob/afc65f9d3045f39cfdb4527e68446ce163fb4e34/src/extension.ts#L556).

**Comment by matangover** [comment]
Thanks, I'll look into it

**Comment by Morikko** [comment]
Thank you for taking the time to look at this. You are right, I used to rely on the daemon with this extension but the daemon had many issues. Since, I reverted back to the Microsoft's extension ("Mypy Type Checker"). Sorry for the disturbance.

According to your testing, it is a mypy daemon issue so I will close this issue anyway.

Note: It was [a known issue on Microsoft's extension](https://github.com/microsoft/vscode-mypy/issues/35)
